dev-resources/template/2_lrs.yml

```diff
@@ -399,15 +416,25 @@ Resources:
                     },
                     "webserver": {
                       "httpHost": "0.0.0.0",
-                      "httpPort": ${InstanceHttpPort},
+                      "httpPort": ${InstanceHttpPort}
```

Comment:
correct me if i'm wrong but isn't this an error now? bad json?

Reply:
haha whoops

Reply:
fixed via https://github.com/yetanalytics/lrsql/pull/276/commits/469608f4a803a816294fb0c541e5480830242289